### DISCUSSION TITLE
Task not found; 21

### DISCUSSION THREAD

**Discussion by Giatroo** [apache/airflow#41281] [Q&A]
Hello everyone!

We're getting the following error:

`{standard_task_runner.py:102} ERROR - Failed to execute job X for task T (Task T not found; 21)`

We're using AirFlow version 2.4.1. Also, we're running the tasks with 3 retries with about 20 seconds of delay between them, but all of them get the same error. However, after I manually retrigger the failed tasks, they work successfully. 

I tried looking for this error, but didn't find any thread or documentation. Does anyone know what this error 21 means? 

Thank you very much!

**Comment by omkar-foss** [comment]
>I tried looking for this error, but didn't find any thread or documentation. Does anyone know what this error 21 means?

@Giatroo 21 is the output of `os.getpid()`, you're getting this log from [here](https://github.com/apache/airflow/blob/2.4.1/airflow/task/task_runner/standard_task_runner.py#L102-L108).

To get to the root cause of this error, you may need to dig further into those logs for any exceptions etc.
